{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2023-0434",
    "ASSIGNER": "security@huntr.dev"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "CWE-20"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://github.com/pyload/pyload/commit/a2b1eb1028f45ac58dea5f58593c1d3db2b4a104",
        "name": "https://github.com/pyload/pyload/commit/a2b1eb1028f45ac58dea5f58593c1d3db2b4a104",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://huntr.dev/bounties/7d9332d8-6997-483b-9fb9-bcf2ae01dad4",
        "name": "https://huntr.dev/bounties/7d9332d8-6997-483b-9fb9-bcf2ae01dad4",
        "refsource": "CONFIRM",
        "tags": [
          "Exploit",
          "Patch",
          "Third Party Advisory"
        ]
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "Improper Input Validation in GitHub repository pyload/pyload prior to 0.5.0b3.dev40."
      }
    ]
  }
}